src/app/Table.elm

```diff
@@ -0,0 +1,29 @@
+module Table exposing (view, init, Msg)
+
+import Html exposing (..)
+
+
+{-
```

Comment:
After the import, it is treated as module documentation, so formatted accordingly and comes up if you want to make it look pretty if you ever publish it. Before a function it becomes function doc.